atomic_cms.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = 'atomic_cms'
-  s.version     = '0.2.3'
+  s.version     = '0.2.4'
   s.date        = '2015-06-19'
```

Comment:
As a side note, it looks like the date isn't required per the [rubygems gemspec doc](http://guides.rubygems.org/specification-reference/)